figure.html

```diff
@@ -207,12 +272,50 @@
 </li>
 
 <li>
-<p class="first">Click on the <span class="bold">Labels</span> tab to add a scale bar and add or edit labels for panels.</p>
-<p>The drop-down menu enables you to use dataset or image name or metadata fields to automatically add labels and colours.</p>
-<p>Label text can also be typed into the box.</p>
-<p>When text, font size, positioning and colour have been selected, click <span class="bold">Add</span>.</p>
-<p>You can change the text or setting on a label at any stage, or delete the label.</p>
+<p class="first">Click on the <span class="bold">Labels</span> tab to add a scale bar and add or edit labels on panels.</p>
+
+<p>Click <span class="bold">Show</span> to add a scale bar to selected images.</p>
 
+<p>If the image does not have pixel sizes in its metadata, you must manually set this before you can add a scale bar.</p>
+
+<p>Use the text box to adjust length and the drop-downs to select position and colour.</p>
+
+<p>Click the check box to add a label and select the font size from the drop down.</p>
+
+<p>As you zoom in on images, scale bars may extend outside the panel.</p>
+
+<p>If this occurs use the scale bar text box to adjust the length of the bar shown.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/figureScalebar.jpg" class="align-center" src="images/figureScalebar.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li>
+<p lass="first">Select from the drop-down menu to use image name, dataset or metadata fields to automatically add labels and colours,</p>
+
+<p>or type your own text into the box.</p>
```

Comment:
This line should be part of the previous paragraph (not a separate <p>). Also, I think you can remove the "and colours". Unless you want to explain the full behaviour of using channel colours for channel labels, but probably best to keep it simple and let users discover the colour behaviour themselves.